src/icpp/canisters/greet/demo-c++17.sh

```
@@ -32,20 +42,21 @@ icpp build-wasm --config icpp-c++17.toml --to-compile all
 echo "--------------------------------------------------"
 echo "Running some manual tests with icp"
-icp canister call greet greet_0 '()' --environment local
-icp canister call greet greet_0_static_lib '()' --environment local
-icp canister call greet greet_1 '()' --environment local
-icp canister call greet greet_2 '("C++ Developer")' --environment local
-icp canister call greet greet_3 '(record { "icpp version" = 1 : int; OS = "Linux" : text })' --environment local
-icp canister call greet greet_4 '(record { 6 = 42 : int; 9 = 43 : int }, record { 7 = 44 : int; 10 = 45 : int })' --environment local
-icp canister call greet greet_json '("{\"name\": \"AJ\"}")' --environment local
-icp canister call greet greet_log_file '()' --environment local
+ID="--environment local --identity $ICPP_PRO_TEST_IDENTITY"
+icp canister call greet greet_0 '()' $ID
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)